packages/relay/src/lib/eth.ts

```diff
@@ -147,10 +147,16 @@ export class EthImpl implements Eth {
   }
 
   private async getFeeWeibars() {
-    let networkFees = await this.mirrorNodeClient.getNetworkFees();
+    let networkFees;
 
-    if (_.isNil(networkFees)) {
+    try {
+      networkFees = await this.mirrorNodeClient.getNetworkFees();
       this.logger.debug(`Mirror Node returned no fees. Fallback to network`);
```

Comment:
This log would also occur if the mirror node did return fees successfully

Reply:
Missed that. Updated